vaadin-dashboard-flow-parent/vaadin-dashboard-flow-integration-tests/src/test/java/com/vaadin/flow/component/dashboard/tests/DashboardDragResizeIT.java

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright 2000-2024 Vaadin Ltd.
+ *
+ * This program is available under Vaadin Commercial License and Service Terms.
+ *
+ * See {@literal <https://vaadin.com/commercial-license-and-service-terms>} for the full
+ * license.
+ */
+package com.vaadin.flow.component.dashboard.tests;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.interactions.Actions;
+
+import com.vaadin.flow.component.dashboard.testbench.DashboardElement;
+import com.vaadin.flow.component.dashboard.testbench.DashboardWidgetElement;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.testbench.TestBenchElement;
+import com.vaadin.tests.AbstractComponentIT;
+
+/**
+ * @author Vaadin Ltd
+ */
+@TestPath("vaadin-dashboard/drag-resize")
+public class DashboardDragResizeIT extends AbstractComponentIT {
```

Comment:
Some tests are failing when running locally due to `MoveTargetOutOfBoundsException: move target out of bounds`. `getDriver().manage().window().setSize(WINDOW_SIZE_LARGE);` in init would help.

But I think the problem is that the page just includes too many widgets and the dashboard not having a fixed size makes it responsive which in turn can cause test failures depending on the window size.

Are all the ITs in this file meaningful for testing the added Flow component logic? I'd suggest simplifying the test page and reducing the number of ITs to only what's necessary.

Reply:
- Reduced the number of widgets on page
- Set a larger window size
- Simplified the set of
  - enlarge horizontally
  - enlarge vertically
  - shrink horizontally
  - shrink vertically
  - enlarge both horizontally and vertically
  - shrink both horizontally and vertically

  to
  - enlarge both horizontally and vertically

  This can be interpreted as any resize and should be enough to cover the feature.